src/autocomplete.mjs

```diff
@@ -243,14 +250,10 @@ export default class extends Controller {
   }
 
   get src() {
-    return this.data.get("url")
+    return this.urlValue
   }
 
   get minLength() {
-    const minLength = this.data.get("min-length")
-    if (!minLength) {
-      return 0
-    }
-    return parseInt(minLength, 10)
+    return this.minLengthValue
```

Comment:
I think we can get rid of `get minLength()` in lieu of using `this.minLengthValue` everywhere?